ipfx/nwb_reader.py

```diff
@@ -429,15 +431,32 @@ def get_nwb_version(nwb_file):
 
 
 def get_nwb1_flavor(nwb_file):
+    """
+    Determine the flavor of nwb file;
+    'Mies':  generated by the MIES hardware with sweeps named as 'data_*'
+    'Pipeline': processed by the pipeline to create epoch information and to rename sweeps as 'Sweep_*'
+    If sweeps are not present, then assume the original 'Mies' format, since no processing had any effect
+
+    Parameters
+    ----------
+    nwb_file: str
+        file name
 
+    Returns
+    -------
+    str
+    """
     with h5py.File(nwb_file, 'r') as f:
         if "acquisition/timeseries" in f:
             sweep_names = [e for e in f["acquisition/timeseries"].keys()]
-            sweep_naming_convention = sweep_names[0].split('_')[0]
+            if sweep_names:
+                sweep_naming_convention = sweep_names[0].split('_')[0]
 
-            if sweep_naming_convention == "Sweep":
-                return "Pipeline"
-            elif sweep_naming_convention == "data":
+                if sweep_naming_convention == "Sweep":
+                    return "Pipeline"
+                elif sweep_naming_convention == "data":
+                    return "Mies"
+            else:
```

Comment:
Could you add a test for the empty nwbv1 file? `test_valid_v1_but_unknown_sweep_naming` can be easily adapted.